src/mkdocs_gallery/plugin.py

```diff
@@ -224,8 +224,8 @@ def on_config(self, config, **kwargs):
 
   # to have the download icons in the buttons
   - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
```

Comment:
It seems that tests break on python 3.7.
Easiest thing to do is to keep retrocompatibility for now.

so I suggest that at the top of the file you insert

```python
import platform
from packaging.version import parse as parse_version

IS_PY37 = parse_version("3.7") <= parse_version(platform.python_version()) < parse_version("3.8")
```

and then line 210 you make `extra_config_yml` a formatted string

```python
mx_name = "materialx" if IS_PY37 else "material.extensions"
extra_config_yml = f"""
...
```

```suggestion
      emoji_index: !!python/name:{mx_name}.emoji.twemoji
      emoji_generator: !!python/name:{mx_name}.emoji.to_svg
```